src/test/java/org/folio/rest/impl/BudgetsTest.java

```diff
@@ -34,4 +36,18 @@ public void testUpdateBudgetWithExceededAllowableAmounts() {
       .body(containsString(ALLOWABLE_ENCUMBRANCE_LIMIT_EXCEEDED.getCode()), containsString(ALLOWABLE_EXPENDITURE_LIMIT_EXCEEDED.getCode()));
   }
 
+  @Test
+  public void testDeleteShouldFailIfThereAreTransactionBoundedToBudget() {
+    logger.info("=== Test Delete of the budget is forbidden, if budget related transactions found ===");
+
+    verifyDeleteResponse(BUDGET.getEndpointWithId(BUDGET_WITH_BOUNDED_TRANSACTION_ID), APPLICATION_JSON, 400).then()
+      .body(containsString(TRANSACTION_IS_PRESENT_BUDGET_DELETE_ERROR.getCode()));
```

Comment:
The test will be more evident if the response is mapped to Errors type. So we make sure it's JSON with needed code, not a simple string with the value you're looking for.

Reply:
Good point. Done